lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
```

Comment:
> So, given all the above should there be unit tests for binding an event on a stream that actually isn't in the pipeline yet? (what's the expected behaviour in that case?)

There's no different code-path for that case, it's all the same to the bind method, so I don't think we'd need to test this; but happy to add one if you think it'd be better?